app/lib/bowling.rb

```diff
@@ -0,0 +1,109 @@
+class Bowling
+  def initialize
+    # => poins contain array with puntuation in game
+    @points = Array.new(10) { nil }
+    
+    # => rounds capture pins downs in all shots, it is matrix 10X2
+    @rounds = Array.new(11) { Array.new(2) { nil } }
+
+    # => turn is a turn played
+    @turn = 0
+  end
+
+  def capture(x, y)
+    begin
+      x = Integer(x)
+      y = Integer(y)
+    rescue
+      return nil
+    end
+    
+
+    if x.negative? || y.negative? || x > 10 || x > 10
+      return nil
+    end
```

Comment:
podrias reducirlo a una linea: 
```suggestion
  return nil if x.negative? || y.negative? || x > 10 || x > 10
```
lo mismo puedes hacer en varios de tus if